### DISCUSSION TITLE
GitHub organizational policies via Terraform

### DISCUSSION THREAD

**Discussion by psub-github** [community/community#35367] [Enterprise]
Hello, We are trying to set the Organizational policies where we can set dependency tracking etc. via TF, is there a way to accomplish that?  Want to automate the organizational dependency tracking via IAC. 

https://docs.github.com/en/enterprise-cloud@latest/admin/policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-dependency-insights-in-your-enterprise.

**Comment by antonblr** [comment]
There is an issue with aggregate list of missing features in terraform provider, and specific comment about org-wide policies -  https://github.com/integrations/terraform-provider-github/issues/379#issuecomment-767296001 
It's getting very low attention, but you'd want to request that feature there.
